advanced_usage/agent-health-check-api.md

```diff
@@ -0,0 +1,36 @@
+# **Agent Health Check API**
+
+**Note: **Using this feature requires atleast version `17.11.0`  of the GoCD server and agent.
+
+The [health check API](https://api.gocd.org/current/#agent-health) allows users to monitor if the agent is connected to the server and is authorized to perform a build.
+
+## Configurations
+
+The API uses the following configurable [agent system properties](https://docs.gocd.org/current/installation/configure-agent-proxy.html#configuring-an-agent-on-linux).
+
+| Property | Values |
+| :---: | :---: |
+| go.agent.status.api.enabled | Defaults to true. Set to false to disable health check api end-point |
+| go.agent.status.api.bind.host | Defaults to localhost.Set to a specific ip address or hostname to bind to that host. Set to 0.0.0.0 to bind to all network interfaces. |
+| go.agent.status.api.bind.port | Defaults to 8152. Set to 0 to use an ephemeral port, which will be displayed in a log statement. |
+
+\*In case the http server is unable to bind \(usually because of port conflicts, or multiple agents running on the same macchine\), a warning will be emitted to the console log and agent startup will continue.
+
+### Configuring Health API for multiple Agents on a Single Machine
+
+In case of multiple agents running on a single machine, the user must explicitly configure the  `go.agent.status.api` properties for each of the agents.
+
+#### On Linux Machine
+
+To configure the properties on linux machine, edit the file corresponding to the agent at /etc/default/go-agent or /&lt;agent-\_working-\_directory&gt;/go-agent.default depending on [agent installation type](https://docs.gocd.org/current/installation/installing_go_agent.html)
```

Comment:
Not sure where you got `/&lt;agent-\_working-\_directory&gt;/go-agent.default` from. Did you find a reference to it in any of the existing docs? That file is not sourced for any of the installations.